source/MRViewer/MRRenderImGuiLabelObject.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include "MRViewer/MRRenderDefaultObjects.h"
+#include "MRViewer/MRUINonOverlappingLabels.h"
```

Comment:
```suggestion
#include "MRRenderDefaultObjects.h"
#include "MRUINonOverlappingLabels.h"
```

Reply:
I prefer full paths because that's what my VSCode autocompletes.